### DISCUSSION TITLE
phishing web site

### DISCUSSION THREAD

**Discussion by cyberturkeyyy** [community/community#136310] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello,

The reported URL address redirects to a phishing website. We kindly request your valuable assistance in closing the address.

https://github.com/sendexee/Sahabet
https://github.com/meritgiris/meritkng
https://github.com/xesmee/Meritking?tab=readme-ov-file
https://github.com/aslanbeygiris/sahabt
https://github.com/gercekgrsx/sahabet
https://github.com/Kralbetx/kralbet?tab=readme-ov-file

PLEASE NOTE THAT: THIS IS URL WORKING ONLY WITH ENTER TURKEY IP'S (TURKEY LOCATIONS USERS) IF YOU WILL ENTER WITH TURKEY LOCATION VPN SERVICE YOU WILL SEE PHISHING CONTENT

Kind Regards

**Comment by cyberturkeyyy** [comment]
I keep reporting a support request, but I've been waiting for 4 days, no response.

**Comment by cyberturkeyyy** [comment]
How can I reach us to the abuse team?

**Comment by cyberturkeyyy** [comment]
We constantly report addresses that lead to such phishing websites, please help us.

**Comment by ebndev** [comment]
Hi @cyberturkeyyy, 👋 thanks for flagging this. I've forwarded this to the proper GitHub Team.

Please note: this community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket and reporting the repositories. For future incidents, please continue to report via the links below.

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

As no other Community Discussions team members will be able to provide additional help with this, I am going to close this discussion. We appreciate your understanding!
